packages/website/src/docs/language-basics/decorators.md

```diff
@@ -32,6 +32,33 @@ The parentheses can be omitted when no arguments are provided.
 model Dog {}
 ```
 
+## Augment decorators
+
+Decorators can also be used from a different location by refering the the type being decoratorated. For this you can declare an augment decorator using the `@@` prefix. The first argument of an augment decorator is the type reference that should be decorated.
+
+```cadl
+model Dog {}
+
+@@tag(Dog, "Sample");
+```
+
+Which is equivalent to
+
+```cadl
+@@tag(Dog, "Sample");
```

Comment:
Wouldn't it be more educational to show equivalence to `@tag("Dog")` here applied directly. Was that the intent?

Reply:
yeah..., I'll update

Reply:
actually this is what its doing no? Both pass "Sample" as the argument.

Reply:
Unless I'm missing something, this currently shows the equivalence of re-ordering the declaration and the augment decorator, but I think you really want to show the equivalence of augment decorator to regular decorator, no?

```suggestion
@tag("Sample")
```

Reply:
Maybe I confused things by writing `@tag("Dog")` where I wanted to write `@tag("Sample")`

Reply:
right fixed it, got tunnel vision on the value